build/Dockerfile

```diff
@@ -191,8 +192,46 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 	&& microdnf clean all
 
 
-############################################# Base image for UBI with NGINX Plus and App Protect WAF/DoS #############################################
-FROM redhat/ubi8 as ubi-plus-nap
+############################################# Base image for UBI with NGINX Plus and App Protect WAF #############################################
+FROM redhat/ubi9 as ubi-plus-nap
+ARG NAP_MODULES
+
+RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode=0644 \
+	--mount=type=secret,id=nginx-repo.key,dst=/etc/ssl/nginx/nginx-repo.key,mode=0644 \
+	--mount=type=secret,id=rhel_license,dst=/tmp/rhel_license,mode=0644 \
+	--mount=type=bind,from=nginx-files,src=nginx_signing.key,target=/tmp/nginx_signing.key \
+	--mount=type=bind,from=nginx-files,src=nginx-plus-9.repo,target=/etc/yum.repos.d/nginx-plus.repo,rw \
+	--mount=type=bind,from=nginx-files,src=app-protect-security-updates.key,target=/tmp/app-protect-security-updates.key \
+	--mount=type=bind,from=nginx-files,src=app-protect-9.repo,target=/etc/yum.repos.d/app-protect-9.repo \
+	source /tmp/rhel_license \
+	## the code below is duplicated from the ubi-plus image because NAP doesn't support UBI 9 and minimal versions
```

Comment:
probably need to audit the commands below as they were added as a stop-gap until ubi9 support was added

Reply:
@vepatel I've updated the comments, from what I can see the changes are still required because it is coming from UBI directly instead of the Plus image we already built. The version part of the comment doesn't really impact as it's the minimal support that causes the need for those lines, but at least now it is explicit about which is supported.